scripts/build/Jenkins/Jenkinsfile

```diff
@@ -1106,15 +1124,6 @@ try {
                         env.REPOSITORY_NAME = repositoryName
                         (projectName, pipelineName) = GetRunningPipelineName(env.JOB_NAME) // env.JOB_NAME is the name of the job given by Jenkins
 
-                        //this can happen on the first run, the pipeline parameters are not yet defined
-                        if(!env.PROJECT_NAME) {
-                            env.PROJECT_NAME = 'AutomatedTesting'
-                        }
-
-                        if(env.PROJECT_NAME != projectName) {
-                            echo "Overriding ProjectName: ${projectName} -> ${env.PROJECT_NAME}"
```

Comment:
Since we're pulling the project name from the pipeline config file now, do we still need the PROJECT_NAME param? Looks like it's no longer used elsewhere in the jenkinsfile.

Reply:
Will actually change this to be an override. Just discussed this with @byrcolin